spot-client/src/spot-tv/app-state/actions.js

```diff
@@ -535,12 +536,17 @@ export function updateSpotTVSource() {
  */
 export function redirectToMeeting(meetingNameOrUrl, { invites, meetingDisplayName, screenshare, startWithVideoMuted }) {
     return (dispatch, getState) => {
+        const state = getState();
+        const domainWhitelist = getMeetingDomainsWhitelist(state);
+
+        isZoomEnabled(state) && domainWhitelist.push('zoom.us');
```

Comment:
Maybe `.*zoom.us`? They support vanity urls.

Reply:
can you give any example? I'm afraid `.*zoom.us` will include malicious URLs like `123zoom.us`

Reply:
`.*.zoom.us` ?

Reply:
ah crap you're right. they support urls like https://1234.zoom.us

Reply:
This makes me wonder if we shouldn't be using regex at all. Instead we should take the domain and split it on period and match based on that? Just do some really dumb, generic string matching.

Reply:
I like the ends in .domain idea. I will try to take a look at least.